### ISSUE TITLE
[BUG] 1.18: Forge And OptiFine seem to be incompatible.

### ISSUE THREAD

**Issue by Motzey187** [sp614x/optifine#6432]
## Description of Issue
Forge instance is crashing when loading up with OptiFine in the mod folder. From the debug-log I could understand that it tried loading up OptiFine first, but then it ends in an "Unexpected issue occured. Exit Code: 0.".

## Steps to Reproduce
1. Download mods in Mod list
2. paste OptiFine and other Mods into the Mods-Folder
3. paste launching Args to the Launching-Argument-Line of the Forge-Profile
4. launch Minecraft
5. game Crashing before showing up.

## Minecraft Launching Arguments (+750MB more RAM = -Xmx2750M )
Launching Args: -Xmx2750M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M 

## OptiFine Version
OptiFine HD U H3 pre6 (most recent) and the release1.18 versions beforehand

## Installation Method
Forge-Mod-Loader

## Forge Version
1.18 - 38.0.14 (most recent) and the release1.18 versions beforehand

## Other Installed Mods
Catalogue:   https://www.curseforge.com/minecraft/mc-mods/catalogue/files/3543625
Configured: https://www.curseforge.com/minecraft/mc-mods/configured/files/3546349
Jade:            https://www.curseforge.com/minecraft/mc-mods/jade/files/3544015
Jade-API:     https://www.curseforge.com/minecraft/mc-mods/jade/files/3544058

## Log Files/Crash Reports
debug-log (versions noted in this bug-report): https://pastebin.com/9J2PWB61
debug-log (versions noted in the pastebin):      https://pastebin.com/SKxdcDvJ

## F3 Debug Screenshot
I am not entering the main menu at all, in this case I sadly can not include that F3-Debug-Screen-Screenshot

## Prior Testing

If you've installed using Forge
- Does this happen without other mods installed? 
   - Yes, it does happen, when OptiFine is being launched without any other mods installed.
   
- Does this happen without any mods at all?
   - There is no crash occurring, if Forge is launched without any mods installed. 

- Does this happen in OptiFine standalone?
   - No. The standalone version is launching up just fine, as expected.

- Does this happen in Vanilla without OptiFine?
   - No. The Vanilla 1.18 us launching without any issues.

- Have you tried using a binary search to find which mods cause this issue?
   - I removed mods one-by-one and in groups (with and without OptiFine):
      https://pastebin.com/T6Fvysjg

## Additional Information
![image](https://user-images.githubusercontent.com/95412753/144716186-4cee5775-f302-4e01-a75e-f56a4c312c74.png)
No crash reports available because of **Exit Code: 0**

## My Conclusion
Forge and OptiFine seem to be incompatible.

**Comment by unAlike** [comment]
I'm just gonna say this now. If you took 2 seconds before posting this you would have seen that this issue is already known about by the developer. He has said it like a hundred times already.

**Comment by Motzey187** [comment]
> I'm just gonna say this now. If you took 2 seconds before posting this you would have seen that this issue is already known about by the developer. He has said it like a hundred times already. Optifine is not currently compatible with forge.

I am sorry, but I couldn't find this post of the developer. But I reported this to Forge-Forums and on here simultaneously. I wanted to have the same report on both pages, so I did report it here as well.

**Comment by Motzey187** [comment]
> I reported this to Forge-Forums

https://forums.minecraftforge.net/topic/106375-release-118-forge-and-optifine-seem-to-be-incompatible

**Comment by fayer3** [comment]
optifine 1.18 is not compatible with forge yet, if you would check the change log it's stated there: https://optifine.net/changelog?f=preview_OptiFine_1.18_HD_U_H3_pre6.jar
if you are on the optifine discord, its also stated int the optifine-updates channel

**Comment by Motzey187** [comment]
@fayer3 
> if you would check the change log it's stated there
I am sorry that I did not look up the changelogs. I often done this beforehand and never had found my issues in them, but on Github, so I skipped the Changelogs with this versions, and directly headed to Github and found nothing about it on here. So I posted it.

> if you are on the optifine discord
I have reached the limit of joined discord servers, and all of them are important, as of now, so I can not join the OptiFine one sadly. I might sort out servers soon though.
